timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -272,6 +272,8 @@ void declare_context(py::module &m) {
 
         F = iterations / store_x_interval
 
+        The first call to `multiple_steps_local` takes longer than subsequent calls, potentials need to be configured internally.
```

Comment:
Usage nit (not for this PR!) -- may be convenient to add a function `ctxt.initialize_local_md()` or similar, that calls these constructors but doesn't perform any steps / doesn't update (x, v, box) state.

(So that this can be done without the small amount of inconsistency that could be introduced by making 1 initial call.)

(And the first call to `multiple_steps_local()` would invoke this, if it hasn't been invoked before.)

Reply:
Definitely. This change would also allow us to avoid the need for the user to pass in a seed every step. Which makes things a bit messy.

We could switch to a more explicit style of calling `ctxt.initialize_local_md` before they can run local md and do all of the parameterization then rather than having to pass in 'good' configurations in each call.